homeassistant/components/tado/entity.py

```diff
@@ -32,6 +32,26 @@ def should_poll(self):
         return False
 
 
+class TadoHomeEntity(Entity):
+    """Base implementation for Tado home."""
+
+    def __init__(self, tado):
+        """Initialize a Tado home."""
+        super().__init__()
+        self.home_name = tado.home_name
+        self.home_id = tado.home_id
+
+    @property
+    def device_info(self):
+        """Return the device_info of the device."""
+        return {
+            "identifiers": {(DOMAIN, f"{self.home_id}")},
```

Comment:
Any reason to use the f-string rather than just reference self.home_id directly?

Reply:
Fixed on latest commit, thanks.